### DISCUSSION TITLE
Version 0.12 Status

### DISCUSSION THREAD

**Discussion by scadergit** [ghettovoice/vuelayers#460] [General]
Bumping to version 12 and noticed it is still being tagged as a release candidate. Thoughts on how much more work is needed to be a normal release, or if there is some other consideration for keeping it a candidate?

**Comment by ghettovoice** [comment]
Hello @scadergit ,
actually v0.12 is ready. Currently only a small bugfixes times to times are detected
But I'm still haven't enough time to finish demo app upgrading and documentation, that I think would be great to do before release.

**Reply by scadergit** [reply]
That's what I figured...would like to help that effort to give back and to learn more. Will probably be similar to your pace for all the same reasons, but I'll work on some pull requests. Do you have any specific page that has been converted that I should model after? If not, I'll start looking over the existing documentation for gaps/changes.

**Reply by ghettovoice** [reply]
I have currently stopped on the demo upgrade step. 
As for the documentation, there is nothing yet changed since v0.11.x

**Reply by ghettovoice** [reply]
As you can see In the `master` branch already sources for v0.12.x, but the `docs` folder have last changes very long time ago

**Reply by ghettovoice** [reply]
Demo app here https://github.com/ghettovoice/vuelayers-demo. And current upgrading work in the `wip-migr` branch.
The app almost functional, however some examples stop working since migration.
